### ISSUE TITLE
New package manager: mip?

### ISSUE THREAD

**Issue by AdamOnLinux** [cbrand/micropython-mdns#10]
Is there a way to install this package via the new package manager mip on the latest micropython 1.19? It no longer uses upip, thanks!

**Comment by cbrand** [comment]
Hey sorry for the late reply.

With the limited knowledge i have about how mip works with an hour I do not think it is feasible to do it:

I did check the mip configuration and it seems like this is something only checking a specific github repository which is handled by the core micropython developers.

According to here: https://github.com/orgs/micropython/discussions/9565 the expected thing to do is to install the files manually if there isn't a mip package (e.g. a reference in the micropython github repo) it is expected that you manually install each file.

I could create a big file with all code in one to make this easier, but do not really see the point there. The issue is that esp32 micropython installs require a custom build anyhow as the MDNS port is otherwise blocked. If you need to do that you should embed the micropython-mdns code base into the image in the first place which results into better performance and less storage use. 

To summarize: mpip does not really seem to support third party libraries and usually if you install the package via mpip or something else, it'll not work as the MDNS port is blocked by the default implementation requiring you to build a custom firmware anyhow.

**Comment by cbrand** [comment]
Hey :) I just did a recheck of the documentation - maybe I missed it or it wasn't there before: https://docs.micropython.org/en/latest/reference/packages.html#writing-publishing-packages

Seems doable, reopening it till I added the necessary config files.

**Comment by cbrand** [comment]
Done and release in version 1.4.0. Install works and tested. See https://github.com/cbrand/micropython-mdns/tree/1.4.0#installation
